lib/views/hunk-view.js

```diff
@@ -102,19 +104,21 @@ class LineView extends React.Component {
     const lineSelectedClass = this.props.isSelected ? 'is-selected' : '';
 
     return (
-      <div
-        className={`github-HunkView-line ${lineSelectedClass} is-${line.getStatus()}`}
-        onMouseDown={event => this.props.mousedown(event, line)}
-        onMouseMove={event => this.props.mousemove(event, line)}
-        onContextMenu={event => this.props.contextMenuOnItem(event, line)}
-        ref={e => this.props.registerLineElement(line, e)}>
-        <div className="github-HunkView-lineNumber is-old">{oldLineNumber}</div>
-        <div className="github-HunkView-lineNumber is-new">{newLineNumber}</div>
-        <div className="github-HunkView-lineContent">
-          <span className="github-HunkView-plusMinus">{line.getOrigin()}</span>
-          <span>{line.getText()}</span>
+      <ContextMenuInterceptor onWillShowContextMenu={event => this.props.contextMenuOnItem(event, line)}>
+        <div
+          className={`github-HunkView-line ${lineSelectedClass} is-${line.getStatus()}`}
+          onMouseDown={event => this.props.mousedown(event, line)}
+          onMouseMove={event => this.props.mousemove(event, line)}
+          onContextMenu={event => this.props.contextMenuOnItem(event, line)}
```

Comment:
I'm not sure so thought I'd ask — can this be removed now that `ContextMenuInterceptor` wraps it?